% \chapter{Three-dimensional assessments are necessary to determine the true, spatially-resolved composition of tissues} \label{chap:chap-2}
% \begin{refsection}
%      % Specify Chapter 2's `.bib` file here
%     \addbibresource{references/chapter2/ref-extracts.bib}
    
%     %%%% OPTIONAL EPIGRAPH EXAMPLE
%     % \epigraph{Do not believe everything you see on the internet.}{-- Albert Einstein}
    
    
%     %%%% MUST: if the chapter is a reprint or submitted paper, you must declare it
%     %% you can use enumerate or itemize environment if you have more than one paper 
%     %% \mybibexclude{} will exclude this citation from the final bibliography
%     %% if this paper appears somewhere else then remove \mybibexclude{} command
    
%     % \begin{singlespace}         % you can also use `onehalfspace` to relax the spacing
%     %     This chapter is adapted from the following article with permission from <publisher>
        
%     %     \fullcite{einstein}. \mybibexclude{einstein}
%     % \end{singlespace} 
    
    
%     %%%% remove the following and add your chapter text here
%     \section{Introduction}
%     Recent developments in spatial profiling technologies have led to the construction of atlases to characterize cellular and tissue compositions, structure, and the “-omic” (genomic, epigenomic, transcriptomic, proteomic, and metabolomic) landscapes of tissues, organs, and whole organisms.1–9 These techniques have led to important discoveries regarding changes in cellular composition during development, aging and the progression of diseases such as cancer and cardiovascular disease. Due to technical and financial limitations, current spatial omic methods are designed to evaluate mm2-sized two dimensional (2D) regions.1,6,10–14 Recently, teams have developed novel techniques, such as open-top light-sheet, micro-CT, serial section based imaging for 3D tissue mapping15–18. However, attempts to quantify the amount of information gained in the transition from 2D to 3D have been limited. The purpose of this manuscript is to interrogate the added value of quantitative 3D pathology over classical 2D analysis. Here, to evaluate the loss in information when comparing inter- and intra-sample tissue heterogeneity, we analyze >100 pancreas tissue samples in the form of tissue cores, whole slide images (WSIs), and cm3-sized 3D samples
%     Consider a histological section of standard, 4 µm thickness, a 1-mm2 core of a tissue microarray (TMA) represents a volume of tissue of just 0.004 mm3, while a common region size for spatial transcriptomics (6.5 x 6.5 mm2) corresponds to a volume of 0.2 mm3. These volumes represent minuscule fractions of the human organs that they are used to represent. More standard techniques, including whole slide images (WSIs) stained with hematoxylin and eosin (H\&E) or immunohistochemistry (IHC), are often considered the gold standard of diagnostic anatomic pathology.19,20 These slides feature a lateral area of 2 x 5 cm2, corresponding to a volume of 5 mm3. The implicit assumption of 2D sampling is that the cells within the sampled region, as well as their morphologies, densities, and cellular and non-cellular neighborhoods, are representative of those of the three-dimensional (3D) organs and diseased tissues from which they are obtained. 
%     Accurate clinical diagnosis of a range of diseases using single 2D H\&E sections (selectively from gross inspection of resected tissues) shows that generalization of findings from 2D is possible, although recent works suggest that relevant criteria including cancer grade and cancer precursor type may be easily misdiagnosed in 2D.21–25 In research settings, where the goal of tissue atlas efforts is generalizability, we hypothesize that 2D sampling may be insufficient to capture the marked intra-sample heterogeneity in cellular composition and tissue architecture.
%     Recent 3D work has demonstrated the utility of tissue clearing and serial sectioning-based approaches to assess microanatomical maps of large (>1 cm3) volumes of tissue at cellular resolution. 17,24,26–38 Here, we use the recently developed 3D imaging workflow CODA to assess the spatial composition of key cells types in thick slabs of both grossly normal human pancreas tissue and human pancreas tissue containing pancreatic ductal adenocarcinoma (PDAC), the deadliest form of pancreatic cancer.17 CODA was recently advanced to enable user interface-guided workflows in an open source programming language39, and has been used to quantitatively interrogate normal human organ development, as well as breast cancer, prostate cancer, pancreatic cancer, diabetic neuropathy, myocarditis, skin regeneration and fetal development in murine and human tissues29,40–48. The uniquely heterogeneous spatial microenvironment of PDAC makes it an optimal testbed to evaluate the benefits of 3D microanatomic mapping over standard 2D approaches.49–51  
%     Our exhaustive analysis demonstrates that standard 2D sampling – using a limited number of TMA cores or WSIs  – is typically insufficient for accurate assessment of tissue composition, tumor content, or the selection of regions of interest for creation of TMA cores and capturing rare events.44 We determine that tens of WSIs and hundreds of TMA cores are necessary to accurately represent the range of tissue compositions present in a cm3-sized human pancreas sample. We find that sections inside a tumor, sometimes just tens of microns apart, can have completely different, uncorrelated cellular and non-cellular structures. Two-dimensional assessments of “representative” slides fail particularly in enumeration of rare events, such as estimation of the density of cancer or cancer precursor cells in samples known to have low neoplastic content.28,44 This work helps clarify the impact of tissue subsampling in study of the composition of normal and malignant tissues, using analysis of 2D and 3D pancreatic human tissue samples as a testbed.  
    
%     \section{RESULTS}
    
%     \subsection{Construction of cohorts of 2D and 3D microanatomically labelled pancreatic tissue}
%     To interrogate the differences between inter-sample and intra-sampled compositional heterogeneity, pancreatic tissue from 149 individuals was retrospectively collected, consisting of 101 samples containing invasive pancreatic cancer and 48 samples containing grossly normal pancreas (Fig 1A). Three cohorts consist of (1) the “2D-WSI” cohort: 64 individual, pathologist-curated WSIs; the “3D-CODA” cohort: fourteen samples containing serially sectioned 3D blocks (seven of which contain invasive pancreatic cancer); and the “TMA” cohort: a single TMA containing pancreas histology from 30 individuals. Cohorts were matched between the TMA, WSI, and 3D cohorts, according to age and gender (Table S1).
%     We used a segmentation algorithm to label microanatomical components to a resolution of 1 µm (See Materials and Methods). Independent testing showed an overall accuracy of 93.2\% across all samples (Fig S1). For the 3D-CODA cohort, image registration was performed to create digital tissue volumes (Fig 1B). The minimum number of sections for these 3D samples was 270 (mean: 297, interquartile range: 816). The median reconstructed volume was 39.0 mm3 (mean: 132.2 mm3, interquartile range: 247.3 mm3). Statistical sampling was conducted on the 2D and 3D cohorts to evaluate the impact of sampling on tissue composition analysis of heterogeneous microanatomical tissue components (Fig 1C).
    
%     \subsection{Spatial correlation rapidly decays within pancreatic tumors}
%     To assess the structural continuity of tissues, we calculated how rapidly tissue composition changed along a straight line through the 3D tumors. To determine the correlation length of each tissue component (PDAC, vasculature, fat, ducts, etc.) – i.e. the distance over which the composition remained significantly correlated – we calculated pixel-to-pixel correlation of these tissue components in 3D (Fig 2A). If this correlation is high, then sampling of a tumor can be sparse. As a limit, if this correlation is perfect, then a 2D section is sufficient to capture the composition of the tumor. 
%     This correlation was calculated for each tissue component and for all whole-slide images spaced between 4 µm and 720 µm apart, averaged across the seven 3D tumor samples and plotted as a function of distance (Fig 2B). Making intuitive sense, our analysis revealed that more abundant structures such as ECM and acini remained spatially correlated over large distances within the blocks, requiring >180 slides (or 720 µm) until they reached a spatial correlation that had decreased by >50\%. For sparser tissues, such as nerves and vasculature, this correlation dropped by >50\% within just 24 µm, or six 4-mm-thick slices (Fig 2C).  Hence tissue composition becomes rapidly decorrelated within pancreatic tumors.
%     To determine whether this rapid decorrelation holds in non-diseased organs, we conducted a similar analysis in seven 3D samples of grossly normal pancreas. Interestingly, the spatial correlation of ECM dropped more rapidly in normal tissue, with loss of >50\% in just 24 µm (compared to 720 µm in cancer tissue). As expected, we found that the spatial correlation in acinar tissue decayed more slowly in normal pancreas, reflecting the marked acinar atrophy and desmoplastic stromal deposition that occurs in pancreatic cancer.
%     We repeated this calculation for samples virtually cut to 6.5 x 6.5 mm2, the area used in some spatial transcriptomics analyses (Fig S2). For tissue components including ducts, PDAC, islets of Langerhans, blood vessels, nerves, and fat, a decrease in spatial correlation of fifty percent was observed within just 40 µm, or 10 sections. 
%     In conclusion, tissue composition changes rapidly in both normal and diseased tissues, highlighting the necessity of 3D assessments to fully capture their spatial organization.
    
%     \subsection{Limitations of core-needle biopsies in assessment of tumor heterogeneity in tissue composition}
%     TMAs cores are often created following pathologist-selected regions of interest (ROIs) on a single histological section that contains a target structure (e.g. cancer). Hundreds of sections may be subsequently cut from these cores for use by researchers who aim to study the original structure chosen by the pathologist. We hypothesized that due to the rapid changes in tissue composition across 3D tumors (Fig. 2), the specific target structures and cellular features selected by pathologists in the initial ROIs could quickly be lost in the cores as successive sections are cut. To quantify this, we created virtual cores within our 3D samples (Fig 3A). We manually chose 50 locations on the first H\&E section of two 3D samples containing visually high cancer content. From these virtual cores, we digitally cut virtual TMA sections (vTMAs) and quantified the change in tissue composition compared to the first (target) section (Fig 3B). 
%     First, we considered the situation where researchers’ objective is to profile the composition of the tumor microenvironment (Fig 3C). We quantified changes in stromal cell density across vTMA sections to assess whether the number and identity of stromal cells would vary greatly between slides, leading to the possibility that two researchers, studying sections from the same TMA cut hundreds of slides apart, could reach opposite conclusions. We found that, as subsequential sections are cut from the initial pathologist-selected ROI, stromal cell density errored on average 25\% within the first 100 sections (0.4 mm), with many simulations nearing 100\% change within 300 sections (~1.2 mm). 
%     Finally, we determined the average number of virtual sections within which virtual cores lost their target structure altogether (Fig 3D). In this case, core ROIs were chosen as containing high cancer content. We thus determined, how many of the 100 simulated cores no longer contained cancer for each virtual section. We found that nearly 50\% of cores contained no cancer within 200 sections (0.8 mm), with this number approaching 75\% after 300 sections. 
%     This analysis demonstrates a rapid decorrelation in cancer content even within expert-guided cores, suggesting that TMA cores may rapidly lose the benefit of expert-guided ROI selection as sections are cut. 
    
    
%     \subsection{Hundreds of TMAs are necessary to capture the true tissue composition of WSIs and 3D tumors}
%     Conventional histological analysis often relies on 2D tissue sections or TMAs to quantify the overall composition of tumors. While practical for large-scale studies, this approach assumes that these limited samples are representative of entire heterogeneous 3D tumors. This can lead to significant loss of information, which can overlook important spatial tissue composition variations and miss rare cell populations. Here, we aimed to quantify information loss when subsampling a heterogeneous 3D sample through 2D histology. To do this, we randomly simulated virtual TMA (vTMAs) with 1 mm diameter in the 2D WSIs and the 3D samples (Fig 4A). We quantified the error in tissue composition for various numbers of random, non-overlapping vTMAs compared to the true, 3D tissue composition (Fig 4B). This process was repeated to quantify the error between vTMAs and 2D WSI composition (Fig 4C), and the error between 2D WSIs and 3D tissues (Fig 4D). 
%     As expected, increasing the number of TMAs taken from a sample decreased the error of estimation of tissue composition of that sample, and this error varied across different microanatomical tissues (Fig 4B, 4C, 4D). By comparing the number of 2D sections necessary to reach <10\% error, we identified tissue components of high and low heterogeneity (Fig 4E, 4F, 4G). We identified ECM as the component with the lowest heterogeneity, with an average of 19 TMAs necessary to reach <10\% error in the estimation of 2D-WSI composition, 22 TMAs necessary for estimation of 3D-volume composition, and only 1 WSI necessary to correctly estimate 3D-volume composition (within 10\% error). In contrast, we identified cancer as a much more heterogeneous structure, with >500 TMAs necessary to estimate the true 3D composition with <10\% error.
%     We repeated this calculation for samples virtually cut to 6.5 x 6.5 mm2, the area often used in spatial transcriptomics analyses (Fig S2).52–55 Our analysis demonstrated that tissue components including acini, islets of Langerhans, PDAC, ducts, blood vessels required roughly 50 simulated sections to estimate true 3D tissue composition with < 10\% error. Overall, this analysis demonstrates that subsampling heterogeneous tumors leads to significant information loss, and that this information loss may be quantified through simulation of 3D anatomical tissue maps.
    
%     \subsection{Sampling guidelines in pancreatic cancer determined through 3D assessment of neoplastic content}
%     In studies of pancreatic cancer initiation and its precursors (PanINs), accurate understanding of the number and composition of cancer precursors in the ductal system is necessary to determine the risk of a given precursor lesion to progress to cancer.56–58 Yet, it is not currently feasible to profile entire human pancreases at cellular resolution to quantify all precursors. Here, we demonstrate that the amount of tissue necessary for incidence profiling may be estimated using simulations of 3D tissue. To do this, we assessed the sampling necessary to reach a preset error in the estimation of neoplastic content. 
%     For this calculation, we utilized a previously reported cohort of 48 large 3D reconstructed samples of human pancreas tissue containing pancreatic intraepithelial neoplasia (PanIN), the precursors to pancreatic cancer5. We defined PanIN burden as the volume percent of PanIN within the pancreatic ductal system. Next, we calculated PanIN burden for all possible combinations of consecutive slides subsampled from 3D and calculated the relative error of the subsampled region to that of the full 3D sample (Fig 5A). Visualizing this as bar plots for low, medium, and high PanIN burden revealed that fewer slides are needed to accurately determine the neoplastic content of samples containing extensive PanIN, while many slides are needed to accurately determine the neoplastic content of samples containing fewer PanIN (Fig 5B). 
%     We conducted the same calculation for cancer content. We defined cancer burden as the percentage of epithelial cells that were classified as PDAC. Again, we found that fewer slides are needed to estimate the composition of cancer in samples with high cancer burden, but that many slides are necessary to estimate cancer composition in samples with low neoplastic content (Fig 5C). 
%     These results suggest the rather intuitive guideline that the rarer the tissue component to be studied is, the larger the number of sections required for a rigorous assessment of that component content. This calculation may be used in the design of studies seeking to minimize the amount of tissue collected for accurate estimation of rare structures. 
    
%     \section{Discussion}
%     Methods for spatially resolved cellular profiling have enabled in-depth quantitative mapping of tissues and tumors to study inter-patient and intra-patient differences in normal human anatomy and disease onset and progression. These methods profile extremely limited regions, which may impact the evaluation of tissue content and local heterogeneity due to tissue sub-sampling. 
%     Here, we used CODA to quantitatively compare inter- and intra- sample heterogeneity through the lens of tissue composition. Using the pancreas as a model system, we demonstrated that correlation of tissue structures decays within tens of microns, even in normal tissues, that the target ROI selected by expert pathologists in design of tissue cores may be rapidly lost within tens of sections, and that tens of WSIs and hundreds of TMAs are needed to recapitulate the true, 3D tissue composition. Further, we demonstrated that quantification of 3D mapped tissues may be used to estimate the number of sections necessary for accurate 2D experimentation. Given that spatial heterogeneity is common in many tumors, similar patterns of rapid tissue structure decay and sampling challenges are also likely to be present in other cancers. To improve tissue analysis accuracy and reduce information loss, it is crucial to account for these subsampling factors.
%     While this work studies inter- and intra- tumoral heterogeneity through meticulous enumeration of tissue composition, it is likely that molecular, genomic, and transcriptomic heterogeneity is also high within 3D tissue samples. For example, recent work has shown that PanIN (precursors to pancreatic cancer), exhibit great inter- and intra- lesional heterogeneity in KRAS mutations, suggesting that these lesions develop primarily from independent genetic events and may meet and merge within the ductal system.5,47 Using CyCIF to measure cellular heterogeneity, groups have also shown the alterations in marker-positive cells from TMAs to WSIs59.
%     The impact of transitioning from 2D to 3D analysis has been shown to be important in clinically relevant features, such as Gleason grade in prostate cancer, which can vary greatly across short distances of a 3D sample.24,28,31,59–65 Additionally, recent work on pancreatic precursor lesions indicates that diagnostic criteria for intraductal papillary mucinous neoplasms (IPMN), primarily based on 2D planar cross-sectional dimensions, may be inaccurate when moving to 3D, potentially needing reclassification of some lesions as pancreatic intraepithelial neoplasias (PanINs)21.
%     Despite the potential of 3D imaging, its widespread adoption in basic and clinical research has been limited by high operational costs and the technical expertise necessary for sample processing, imaging, and computational analysis. However, several technological developments are reducing these barriers. The decreasing cost of chemical reagents for optical tissue clearing66–70, optimized software71,72, and the integration of generative artificial intelligence for interpolation of missing or damaged images in volumetric stacks are collectively enhancing scalability73. In parallel, the emergence of open-source GUIs for light-sheet fluorescence microscopy and serial section histology reconstruction has accelerated the use of 3D in tissue analysis in biomedical and oncologic research39,74.
%     In sum, we demonstrate in this work that 3D assessments are necessary to accurately assess tissue composition and tumor content and provide guidelines for the rate of sampling necessary to rigorously assess spatially resolved tissue composition and associated tissue density and intercellular distances.
    

%     \printbibliography[heading=subbibliography, title={References}]
% \end{refsection}    

\chapter{Three-dimensional assessments are necessary to determine the true, spatially-resolved composition of tissues}
\label{chap:chap-2}

\begin{refsection}
    % Set up reference file for Chapter 2
    \addbibresource{references/chapter2/ref-extracts.bib}
    
    % Chapter Text
    \section{Introduction}
    Recent developments in spatial profiling technologies have led to the construction of atlases to characterize cellular and tissue compositions, structure, and “omic” landscapes (genomic, epigenomic, transcriptomic, proteomic, metabolomic) of tissues, organs, and even whole organisms \cite{Liu2020High,Ozsolak2011RNA}. These techniques have unveiled crucial insights regarding changes in cellular composition during development, aging, and the progression of diseases such as cancer and cardiovascular conditions. 

    Due to technical and financial constraints, most spatial omic methods are limited to evaluating 2D regions of tissue, typically on the scale of \( \text{mm}^2 \) \cite{ref10, ref14}. However, innovative techniques such as open-top light-sheet microscopy, micro-CT, and serial sectioning-based imaging are now emerging to enable 3D tissue mapping \cite{ref15, ref18}. Despite these advances, the quantitative benefits of transitioning from 2D to 3D remain inadequately explored. The purpose of this chapter is to evaluate how 3D quantitative pathology adds value over classical 2D approaches, particularly for assessing inter- and intra-sample heterogeneity in >100 pancreas tissue samples.

    For instance, a tissue core of standard 4-\( \mu \text{m} \) thickness represents just \( 0.004 \, \text{mm}^3 \) of tissue volume in a 1-\( \text{mm}^2 \) microarray core, while the larger regions commonly evaluated in spatial transcriptomics (\( 6.5 \times 6.5 \, \text{mm}^2 \)) represent only \( 0.2 \, \text{mm}^3 \) \cite{ref13}. Whole slide images (WSIs) and other 2D sampling methods are considered the gold standard in diagnostic pathology, but they assume that these thin 2D sections adequately represent the 3D structures of the tissues they originate from.

    Recent studies suggest this assumption may not hold, particularly in cancer research, where critical diagnostic markers such as tumor grade or precursor lesion type could be easily misrepresented in 2D \cite{ref21, ref25}. For a fuller understanding, 3D analyses—such as the CODA pipeline—are essential for assessing spatial organization and resolving intra-sample heterogeneity in both normal and malignant pancreatic tissues \cite{ref28}.

    \section{Results}

    \subsection{Spatial correlation rapidly decays within pancreatic tumors}
    To quantify spatial continuity, we assessed how tissue composition changes as a function of distance within 3D pancreatic tumors. The correlation length for each tissue component (e.g., cancer cells, vasculature, fat, ducts) was determined by calculating pixel-to-pixel correlations in all seven 3D tumor samples (Fig. 2A).

    Our analysis revealed that abundant structures like the extracellular matrix (ECM) and acini maintained correlations over larger distances, requiring more than 720-\( \mu \text{m} \) to lose 50\% correlation (Fig. 2B). In contrast, sparse structures such as nerves and blood vessels decorrelated rapidly, with a 50\% drop occurring within just 24-\( \mu \text{m} \) (Fig. 2C). Additionally, non-diseased pancreas tissues showed even faster correlation loss for structures like ECM, likely reflecting the stromal and acinar atrophy often observed in pancreatic cancer.

    When virtually cutting regions corresponding to \( 6.5 \times 6.5 \, \text{mm}^2 \), the spatial correlation for key components (e.g., ducts, cancer, islets of Langerhans, blood vessels) dropped below 50\% in just 40-\( \mu \text{m} \), highlighting the need for 3D assessments.

    \subsection{Limitations of core-needle biopsies in assessing tumor heterogeneity}
    Tissue microarray (TMA) cores are widely used to study tumor composition, but they assume selected 2D cores remain representative as additional sections are cut. To test this, we digitally simulated sequential virtual TMAs (vTMAs) from 3D samples, starting from pathologist-selected regions of interest (ROIs) (Fig. 3A). Our analysis showed that after cutting just 200 sections (~0.8 mm), 50\% of virtual cores no longer contained cancer (Fig. 3D). This highlights the potential for loss of diagnostic accuracy when TMAs are used to analyze heterogeneous tumors.

    \subsection{Hundreds of TMAs are necessary to capture the true tissue composition of WSIs and 3D tumors}
    To quantify the information loss caused by subsampling, we simulated random vTMAs for different tissue components and calculated the error in tissue composition. While some components (e.g., ECM) could be accurately estimated with minimal subsampling, hundreds of TMAs were needed to estimate tumor heterogeneity within 10\% error (Fig. 4).

    \section{Discussion}
    Spatial profiling technologies provide valuable insights into tissue architecture and pathology, but their limitations in 2D sampling remain an obstacle for accurately capturing the complexities of 3D tissues. Our analysis demonstrates the necessity of 3D assessments for both diagnostic and research applications. Using CODA, we quantified sampling biases and identified that 2D sections often fail to represent the distribution of sparse or heterogenous tissue components.

    Advances in tissue clearing, computational algorithms, and cost reduction are gradually lowering barriers for implementing 3D imaging in routine tissue analysis \cite{ref66, ref74}. These developments, combined with open-source tools and artificial intelligence, are likely to accelerate the adoption of 3D methods in biomedical research.

    In summary, our findings argue for the incorporation of 3D assessments in studies of cancer biology and tissue heterogeneity, offering improved accuracy and the ability to characterize rare events and compositional variation with greater fidelity.

    \printbibliography[heading=subbibliography, title={References}]
\end{refsection}